pkg/relayer/session/session.go

```diff
@@ -653,6 +667,34 @@ func (rs *relayerSessionsManager) deleteSessionTrees(
 	)
 }
 
+// deleteSession deletes the session tree from the relayerSessions and
+// removes it from the disk store.
+func (rs *relayerSessionsManager) deleteSession(sessionTree relayer.SessionTree) {
```

Comment:
I think we should EITHER:
- rename this to `#deleteSessionTree()` _(my preference)_
- rename `#deleteSessionTrees()` to `#deleteSessions()`

```suggestion
// deleteSessionTree deletes the session tree from the relayerSessions and
// removes it from the disk store.
func (rs *relayerSessionsManager) deleteSessionTree(sessionTree relayer.SessionTree) {
```